erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -182,6 +185,11 @@ def update_package_tag(self, reset=False):
 					})
 					package_tag.save()
 
+	def set_per_picked(self):
+		picked_qty = sum(d.picked_qty for d in self.locations)
+		ordered_qty = sum(d.qty for d in self.locations)
+		per_picked = (picked_qty / ordered_qty) * 100
+		frappe.db.set_value("Sales Order", self.locations[0].sales_order, "per_picked", per_picked)
```

Comment:
Avoid doing this using `set_value`, set it in the doc and save it instead.

Also, Humboldt's flow usually uses only 1 order, but our system should be able to handle multiple orders.

Break this down as:

```python
for order in all_orders:
	picked_qty = sum(picked_qty)
	ordered_qty = sum(ordered_qty)
	per_picked = (picked_qty / ordered_qty) * 100

	order = frappe.get_doc(order)
	order.per_picked = per_picked
	order.save()
```